### ISSUE TITLE
Kubefirst API fails to start after restarting k3d cluster

### ISSUE THREAD

**Issue by gregory-j-baker** [kubefirst/kubefirst#1836]
### Which version of kubefirst are you using?

2.2.17

### Which cloud provider?

k3d (local)

### Which DNS?

None specific

### Which installation type?

CLI

### Which distributed Git provider?

None specific

### What is the issue?

The kubefirst-api fails to start after k3d restarts, giving the following error:

```
time="2023-10-13T10:14:01Z" level=fatal msg="error connecting to mongodb: connection() error occurred during connection handshake: auth error: unable to authenticate using mechanism \"SCRAM-SHA-256\": (AuthenticationFailed) Authentication failed."
```

MongoDB logs the following:

```
{"t":{"$date":"2023-10-13T10:29:50.615+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:48366","uuid":"918d95b9-1940-4e6d-b375-aa761c979904","connectionId":1,"connectionCount":1}}
{"t":{"$date":"2023-10-13T10:29:50.623+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn1","msg":"client metadata","attr":{"remote":"127.0.0.1:48366","client":"conn1","doc":{"application":{"name":"mongosh 1.8.2"},"driver":{"name":"nodejs|mongosh","version":"5.3.0|1.8.2"},"platform":"Node.js v16.19.1, LE","os":{"name":"linux","architecture":"x64","version":"6.5.6-200.fc38.x86_64","type":"Linux"}}}}
{"t":{"$date":"2023-10-13T10:29:51.735+00:00"},"s":"I",  "c":"-",        "id":20883,   "ctx":"conn1","msg":"Interrupted operation as its client disconnected","attr":{"opId":140}}
```

Both the DB and the API are using the same secret for the password: `secretKeyRef(kubefirst-initial-secrets.mongodb-root-password)`. 🤷

## Steps to reproduce

- `kubefirst k3d create` (verify everything works as expected)
- `k3d cluster stop kubefirst`
- `k3d cluster start kubefirst`

**Note:** everything else in the cluster appears to be working fine.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fharper** [comment]
Thanks for reporting the issue @gregory-j-baker . Sorry about that, it is [fixed](https://github.com/kubefirst/charts/pull/140/files) and will be available in the next release. 2.3 should go out by the end of the week.

Let us know if the new version don't fix the issue please.

**Comment by gregory-j-baker** [comment]
I tried to verify the fix tonight, but ran into different issues (see <https://github.com/kubefirst/kubefirst/issues/1874>).

**Comment by gregory-j-baker** [comment]
I can confirm this issue is fixed in 2.3.x! 🥳
